plugins/filter/json_diff.yml

```
@@ -0,0 +1,63 @@
+---
+# Copyright (c) Stanislav Meduna (@numo68)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION:
+  name: json_diff
+  short_description: Create a JSON patch by comparing two JSON files
+  description:
+    - This filter compares the input with the argument and computes a list of operations
+      that can be consumed by the P(community.general.json_patch_recipe) to change the input
+      to the argument.
+  requirements:
+    - jsonpatch (on the controller)
```

The comment on controller isn't needed since it is added automatically, see https://docs.ansible.com/ansible/devel/collections/community/general/jc_filter.html#requirements: "The below requirements are needed on the local controller node that executes this filter."
```suggestion
    - jsonpatch
```